tests/components/Pagination.test.tsx

```diff
@@ -0,0 +1,148 @@
+import { render, screen, within } from '@testing-library/react';
+import { VerticalResults, VerticalResultsProps } from '../../src/components/VerticalResults';
+import { State, VerticalSearchState } from '@yext/answers-headless-react';
+import { StandardCard } from '../../src/components/cards/standard/StandardCard';
+import { spyOnActions, mockAnswersState, mockAnswersHooks } from '../__utils__/mocks';
+import { mockedVerticalResults } from '../__fixtures__/data/verticalresults';
+import userEvent from '@testing-library/user-event';
+
+const mockedState: Partial<State> = {
+  vertical: {
+    verticalKey: 'vertical',
+    results: mockedVerticalResults,
+    resultsCount: mockedVerticalResults.length,
+    offset: 0
+  },
+  searchStatus: {
+    isLoading: false
+  },
+  meta: {
+    searchType: 'vertical'
+  },
+  query: {}
+};
+
+const mockedActions = {
+  state: mockedState,
+  setOffset: jest.fn(),
+  executeVerticalQuery: jest.fn()
+};
+
+jest.mock('@yext/answers-headless-react');
+
+beforeEach(() => {
+  mockAnswersHooks({ mockedState, mockedActions });
+});
+
+it('Don\'t display pagination component when allowPagination is false', () => {
```

Comment:
nit: I think the description here and below reads a little better with "doesn't" instead of "don't"

Reply:
renamed tests for fit 'it' sentence structure